protocols/servlet/src/main/java/org/jboss/arquillian/protocol/servlet/ServletMethodExecutor.java

```diff
@@ -75,16 +76,18 @@ public TestResult invoke(final TestMethodExecutor testMethodExecutor) {
         URI targetBaseURI = uriHandler.locateTestServlet(testMethodExecutor.getMethod());
 
         Class<?> testClass = testMethodExecutor.getInstance().getClass();
-        final String url = targetBaseURI.toASCIIString() + ARQUILLIAN_SERVLET_MAPPING
-            + "?outputMode=serializedObject&className=" + testClass.getName() + "&methodName="
-            + testMethodExecutor.getMethod().getName();
-
-        final String eventUrl = targetBaseURI.toASCIIString() + ARQUILLIAN_SERVLET_MAPPING
-            + "?outputMode=serializedObject&className=" + testClass.getName() + "&methodName="
-            + testMethodExecutor.getMethod().getName() + "&cmd=event";
 
         Timer eventTimer = null;
         try {
+            String urlEncodedMethodName = URLEncoder.encode(testMethodExecutor.getMethodName(), "UTF-8");
```

Comment:
I've moved this block of code inside of the try/catch because `URLEncoder.encode()` throws `UnsupportedEncodingException`.